d2bs/kolbot/libs/common/Pather.js

```diff
@@ -861,6 +861,51 @@ ModeLoop:
 
 		return targetArea ? me.area === targetArea : me.area !== preArea;
 	},
+	
+	
+	/*
+	Pather.getAct(targetArea);
+	targetArea - id of the area to enter
+	*/
+	getAct: function getAct(targetArea) {
+		var actNumber,
+			lo,
+			hi,
+			areas = [39, 74, 102, 108, 132];
+		
+
+		hi = areas.length;
+		lo = 0;
+		
+		while (hi >= lo) {
+			actNumber = ((hi - lo) / 2|0) + lo;
+			
+			if (targetArea <= areas[actNumber]) {
+				hi = actNumber - 1;
+			} else {
+				lo = actNumber + 1;
+			}
+			
+		}
+		
+
+		return lo + 1;
+	},
+	
+	
+	/*
+	Pather.broadcastIntent(targetArea);
+	targetArea - id of the area to enter
+	*/
+	broadcastIntent: function broadcastIntent(targetArea) {
```

Comment:
```javascript
broadcastIntent: function (targetArea) {
        if (Config.MFLeader) {
            var targetAct = this.getAct(targetArea);

            if (me.act !== targetAct) {
                say("goto A" + targetAct);
            }
        }
    },
```
Might be better to only check the targetAct and myAct only if MFLeader is set

Reply:
agreed. I will change it